lib/api/kuzzle.js

```diff
@@ -143,7 +144,7 @@ class Kuzzle extends EventEmitter {
           maxMinTerms: this.config.limits.subscriptionMinterms
         });
 
-        return this.repositories.init();
+        return true;
```

Comment:
Why return a boolean?
You should instead return `this.validation.curateSpecification()`, removing one `.then`  level